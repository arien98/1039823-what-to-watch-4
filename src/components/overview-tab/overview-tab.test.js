import React from "react";
import renderer from "react-test-renderer";
import OverviewTab from "./overview-tab.jsx";

describe(`main renders correctly`, () => {
  test(`it renders correctly`, () => {
    const props = {
      ratingScore: 8,
      ratingLevel: `Good`,
      ratingCount: 200,
      description: `Early in World War II, an inexperienced U.S. Navy captain must lead an Allied convoy being stalked by Nazi U-boat wolfpacks.`,
      director: `Rian Johnson`,
      starring: [
        `Anthony Mann`,
        `Guy Ritchie`,
      ],
    };

    const tree = renderer
      .create(<OverviewTab {...props}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
