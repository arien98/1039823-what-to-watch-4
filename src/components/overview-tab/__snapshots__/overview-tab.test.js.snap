// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main renders correctly it renders correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        200
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      Early in World War II, an inexperienced U.S. Navy captain must lead an Allied convoy being stalked by Nazi U-boat wolfpacks.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Rian Johnson
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Anthony Mann, Guy Ritchie
        and other
      </strong>
    </p>
  </div>,
]
`;
